src/components/inputsList/InputsList.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import Input from '../input/Input';
+
+export default function InputsList({ inputs }) {
```

Comment:
this is "dumb" presentational component, so it should accept an array of inputs.
computations like this:
```javascript
 Object.keys(inputs).map((inputId) => (..)
```
should be done in mapStateToProps function.
This component should not be tied to redux.
In your implementation it knows about state shape. Dumb components should be able to work in any environment which provide an array of inputs.